.eslintrc.json

```
@@ -13,7 +13,9 @@
   },
   "rules": {
     "@typescript-eslint/no-unused-vars": 2,
-    "indent": ["error", 2],
+    "indent": ["error",
+      2,
+      { "SwitchCase": 1 }],
```

❗️
Даже если prettier автоматически не подхватывает такие кейсы, их можно отредачить руками и написать удобнее
```suggestion
    "indent": [
      "error",
      2,
      { "SwitchCase": 1 }
    ],
```